src/include/wiredtiger.in

```diff
@@ -4685,15 +4685,20 @@ extern int wiredtiger_extension_terminate(WT_CONNECTION *connection);
 #define	WT_STAT_DSRC_BLOCK_MINOR			2021
 /*! btree: btree checkpoint generation */
 #define	WT_STAT_DSRC_BTREE_CHECKPOINT_GENERATION	2022
-/*! btree: column-store fixed-size leaf pages */
+/*! btree: column-store fixed-size leaf pages, only reported if
+ * statistics=all is set */
```

Comment:
@daveh86 was right here. It would be better if these comments are block comments to match for example: https://github.com/wiredtiger/wiredtiger/blob/develop/src/include/wiredtiger.in#L220

Ping me if it's more than an hour worth of work to update the script to generate comments in the correct form.